Makefile

```diff
@@ -20,7 +20,7 @@ clean:
 .PHONY: build
 build:
 	mkdir -p ${DATAPLANEAPI_PATH}/build
-	CGO_ENABLED=$(CGO_ENABLED) go build -ldflags "-X main.GitRepo=${GIT_REPO} -X main.GitTag=${GIT_LAST_TAG} -X main.GitCommit=${GIT_HEAD_COMMIT} -X main.GitDirty=${GIT_MODIFIED} -X main.BuildTime=${BUILD_DATE}" -o ${DATAPLANEAPI_PATH}/build/dataplaneapi ${DATAPLANEAPI_PATH}/cmd/dataplaneapi/
+	GOOS=linux CGO_ENABLED=$(CGO_ENABLED) go build -ldflags "-X main.GitRepo=${GIT_REPO} -X main.GitTag=${GIT_LAST_TAG} -X main.GitCommit=${GIT_HEAD_COMMIT} -X main.GitDirty=${GIT_MODIFIED} -X main.BuildTime=${BUILD_DATE}" -o ${DATAPLANEAPI_PATH}/build/dataplaneapi ${DATAPLANEAPI_PATH}/cmd/dataplaneapi/
```

Comment:
could you change this to behave the same way as CGO_ENABLED ?
so we allow this to be overridden with ENV var

Reply:
Done